pkg/release/images.go

```diff
@@ -239,6 +239,89 @@ func (i *Images) Validate(registry, version, buildPath string) error {
 	return nil
 }
 
+// Exists that image manifests exist on a specified remote registry.
```

Comment:
I guess the intention is to keep the convention to start the docs with the method name, for example:
```suggestion
// Exists verifies that an image manifests exist on a specified remote registry.
```

Reply:
Yep, that was my goal! And then I threw in some bad grammar to make it interesting :sweat_smile: